src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel.mdx

```diff
@@ -18,19 +18,14 @@ import k8sOTelDiagram from 'images/infrastructure_diagram_k8s-otel-stack.webp'
+OpenTelemetry observability for Kubernetes offers users comprehensive, open-source instrumentation paired with first-class Kubernetes UI that our proprietary Kubernetes instrumentation already supports. Our K8s UIs  UIs are designed to be provider agnostic, giving users complete flexibility to choose between OpenTelemetry or New Relic instrumentation based on their needs.
 
-Once you've got the [Kubernetes integration up and running](/install/kubernetes/), you can start monitoring all the services running on Kubernetes in one place by [exploring your Kubernetes cluster](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/). It shows you everything from the top of the control plane down to the applications running on a single pod, so you can get to the heart of how your cluster works.
 
-    <img
-    title="K8s OpenTelemetry diagram"
-    alt="K8s OpenTelemetry diagram"
-    src={k8sOTelDiagram}
-    />
+This document outlines the process for monitoring a Kubernetes cluster using OpenTelemetry. It involves the installation of the [`nr-k8s-otel-collector`](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector) Helm chart within the cluster and the deployment of the necessary collectors to enable first-class observability.
+
 
-By integrating Kubernetes components into the OpenTelemetry collector, we can send metrics, events, and logs directly to New Relic. These telemetry signals automatically enhance our out-of-the-box experiences such as the [Kubernetes navigator](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#navigator-preview), [overview dashboard](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#cluster-overview-dashboard), [Kubernetes events](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#browse-your-kubernetes-events), or [Kubernetes APM summary page](/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page/). 
+By integrating Kubernetes components into the OpenTelemetry collector, we can transmit metrics, events, and logs directly to New Relic. These telemetry signals automatically enhance our out-of-the-box experiences such as the [Kubernetes navigator](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#navigator-preview), [overview dashboard](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#cluster-overview-dashboard), [Kubernetes events](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#browse-your-kubernetes-events), or [Kubernetes APM summary page](/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page/).
```

Comment:
Suggestion (My POO): in stead of mentioning **we** or **our**, use **You** or **Your**. This empowers the user to directly take actions...For example, By integrating Kubernetes components into the OpenTelemetry collector, **you** can transmit metrics, events,..... These telemetry signals automatically enhance **your**...

Here mentioning **we** means New Relic ...

Reply:
@cbehera-newrelic, thank you for the suggestion, but I think I'll leave it as it is for now because I'm not sure about it. The PM wrote this, so I'll have a chat with him and see if we can make a change.